src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -37,7 +37,7 @@ public partial struct ImmutableArray<T> : IReadOnlyList<T>, IList<T>, IEquatable
         /// <summary>
         /// An empty (initialized) instance of <see cref="ImmutableArray{T}"/>.
         /// </summary>
-        public static readonly ImmutableArray<T> Empty = new ImmutableArray<T>(new T[0]);
+        public static readonly ImmutableArray<T> Empty = new ImmutableArray<T>(Array.Empty<T>());
```

Comment:
Does this compile?  I thought we built this library against a contract that lacks Empty.  And since it's being cached into a singleton in a static, caching it separately isn't really warranted.